bot/exts/utils/extensions.py

```diff
@@ -1,7 +1,7 @@
 import functools
 import logging
-import typing as t
 from enum import Enum
+from typing import Mapping, Optional
```

Comment:
`Mapping` should be imported from `collections.abc` since it's deprecated in `typing`.